test_runner/src/main/kotlin/ftl/util/Utils.kt

```diff
@@ -38,32 +40,21 @@ fun join(first: String, vararg more: String): String {
         .replace(regex = Regex("/+"), replacement = "/")
 }
 
-fun sleep(seconds: Long) {
-    try {
-        Thread.sleep(ofSeconds(seconds).toMillis())
-    } catch (e: Exception) {
-        System.err.println(e)
-    }
-}
-
 // marked as inline to make JaCoCo happy
 @Suppress("NOTHING_TO_INLINE")
 inline fun fatalError(e: Exception, message: String? = null) {
     throw RuntimeException(message ?: e.toString(), e)
```

Comment:
Is was intentional to leave that name unchanged, in opposite to function below?

Reply:
Yes it was, firstly I wanted to get rid of overloaded methods. And secondly I noticed that this method is used for FTL/gcloud related exceptions and renamed one for flank related errors. I wanted to keep it separate.

Reply:
should we rename this to `firebaseFatalError` then to differentiate from `flankFatalError`?

Reply:
I appreciate dedicated exceptions in FlankException.kt this is clean and meaningful but `fatalError` is legacy and now looks kinda odd for me. Maybe te best solution will be getting rid of that helper functions and replace them with direct throws?

Reply:
The only issue I have with direct throws is it's not clear why we're throwing. A helper may communicate better?

Reply:
Hmm, I guess that the type of exception plus additional message can communicate as well. Helper adds one additional step to stack trace which holds source of throw.

Reply:
> the type of exception plus additional message can communicate as well. Helper adds one additional step to stack trace which holds source of throw.

That makes sense. I'm fine with dropping the helper.

Reply:
@bootstraponline @jan-gogo 
Hey! As suggested (and I agree with your comments) I removed helper methods